authutils/auth_service_test.go

```diff
@@ -4,6 +4,7 @@ package authutils
 import (
 	"github.com/magiconair/properties/assert"
```

Comment:
Why use two different `assert` libraries here: lets remove it and keep `github.com/stretchr/testify/assert`